docs/v3/source/includes/concepts/_includes.md.erb

```diff
@@ -25,7 +25,7 @@ Example request to apps resource to include parent orgs and spaces
 ```
 
 ```shell
-curl "https://api.example.org/v3/apps?include=org,space" \
+curl "https://api.example.org/v3/apps?include=space.organization" \
```

Comment:
I think this section of the docs is supposed to demo including multiple resources. Should this be:

```
curl "https://api.example.org/v3/apps?include=space,space.organization" \
```

Reply:
`space.organization` will automatically include spaces also. This is because without the space, there will not be a way to determine which organization is associated with which app